master/internal/experiment.go

```diff
@@ -191,8 +191,8 @@ func (e *experiment) Receive(ctx *actor.Context) error {
 			e.restoreTrialsFromPriorOperations(ctx, e.searcher.TrialOperations)
 		} else {
 			e.processOperations(ctx, e.searcher.TrialOperations, nil)
+			ctx.Tell(e.hpImportance, hpimportance.ExperimentCreated{ID: e.ID})
```

Comment:
might as well fix it in both places. only send on creation not restore and create the manager sooner.